apps/judicial-system/web/src/components/TagAppealState/TagAppealState.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+
+import {
+  CaseAppealRulingDecision,
+  CaseAppealState,
+} from '@island.is/judicial-system/types'
+
+import { appealRuling } from '@island.is/judicial-system-web/messages/Core/appealRuling'
+import { Tag, TagVariant } from '@island.is/island-ui/core'
+import { tables } from '@island.is/judicial-system-web/messages'
+
+interface Props {
+  appealState?: CaseAppealState
+  appealRulingDecision?: CaseAppealRulingDecision
+}
+
+const TagAppealState: React.FC<Props> = ({
+  appealRulingDecision,
+  appealState,
+}) => {
+  const { formatMessage } = useIntl()
+
+  if (!appealRulingDecision) return null
```

Comment:
This means that a tag is only shown when an appeal ruling decision has been entered. Newly appealed and received appeals will not get a tag. The code will work if this line is removed.

Reply:
You are right I just forgot to remove this! 🙈